src/Image.js

```diff
@@ -6,49 +6,49 @@ import { bsClass, prefix, splitBsProps } from './utils/bootstrapUtils';
 
 const propTypes = {
   /**
-   * Sets image as responsive image
+   * Sets image as fluid image.
    */
-  responsive: PropTypes.bool,
+  fluid: PropTypes.bool,
 
   /**
-   * Sets image shape as rounded
+   * Sets image shape as rounded.
    */
   rounded: PropTypes.bool,
 
   /**
-   * Sets image shape as circle
+   * Sets image shape as circle.
    */
-  circle: PropTypes.bool,
+  roundedCircle: PropTypes.bool,
 
   /**
-   * Sets image shape as thumbnail
+   * Sets image shape as thumbnail.
    */
   thumbnail: PropTypes.bool
 };
 
 const defaultProps = {
-  responsive: false,
+  fluid: false,
   rounded: false,
-  circle: false,
+  roundedCircle: false,
   thumbnail: false
 };
 
 class Image extends React.Component {
   render() {
     const {
-      responsive,
+      fluid,
       rounded,
-      circle,
+      roundedCircle,
       thumbnail,
       className,
       ...props
     } = this.props;
     const [bsProps, elementProps] = splitBsProps(props);
```

Comment:
Lets use the newer `createBootstrapComponent` recently added, and no bootstrapUtils. You can check out Button.js for a reference on how to use it!